src/Tasks/Build/File.php

```diff
@@ -91,10 +91,7 @@ public function run()
         }
 
         // Build media (relative path)
-        $media = $this->buildMedia("media/com_" . $this->getExtensionName(), 'com_' . $this->getExtensionName());
-        $media->run();
-
-        $this->addFiles('media', $media->getResultFiles());
+        $this->buildMediaFiles('fil_' . $this->getExtensionName());
```

Comment:
I would not include media (or any other additional) files for `files` type, since it's normally only a single file or folder you want to copy somewhere.

Reply:
Indeed. I removed it.